MapboxNavigation/NavigationMapView.swift

```diff
@@ -197,7 +209,7 @@ open class NavigationMapView: MGLMapView {
         return MGLShapeCollectionFeature(shapes: [baseLine] + lines)
     }
     
-    func shape(for waypoints: [Waypoint]) -> MGLShape? {
+    func shapes(for waypoints: [Waypoint]) -> MGLShape? {
```

Comment:
The name of this method implies that it returns an array of set of MGLShape objects. However, it returns only a single MGLShape object.